index.js

```diff
@@ -171,4 +171,26 @@ function setScale(keypath, width, height) {
 
 function setRotation(keypath, degree) {
   rlottieHandler.rlottieModule[0].lottieHandle.setRotation(keypath, degree);
+}
+
+function onResizeSliderDrag(value) {
+  var width = document.getElementById("player").clientWidth;
+  var height = document.getElementById("player").clientHeight;
+  width = width / 4 * 3;
+  height = height / 4 * 3;
+
+  var size = width < height ? width : height;
+  size = size-8;
```

Comment:
What is the meaning of 8 ?
Please keep coding style, please use ```size = size - 8``` instead of ```size = size-8```

Reply:
I deleted this particular line